delta/delta_1.4.0_swmm_views.sql

```diff
@@ -1,721 +1 @@
-DROP SCHEMA IF EXISTS qgep_swmm CASCADE;
```

Comment:
This is not mandatory to remove, but I see no objection to remove it

Reply:
Yes, it wasn't totally clear for me what we should do with these statements (also left in quite a few `DROP ... IF EXISTS` that we could get rid of theoretically since now done in pre-all...